### PR TITLE
Update PDO EventStore implemenations

### PR DESCRIPTION
**Pull request by prolic**

### CODE REVIEW

**Comment by malukenho** [comment]
@prolic add `pdo` to `require` section on `composer.json`.

File: src/Exception/ExtensionNotLoaded.php [hunk 1]
```diff
@@ -10,19 +10,16 @@
 
 declare(strict_types=1);
 
-namespace Prooph\EventStore\PDO\JsonQuerier;
+namespace Prooph\EventStore\PDO\Exception;
 
-use Prooph\EventStore\PDO\JsonQuerier;
-
-final class Postgres implements JsonQuerier
+class ExtensionNotLoaded extends RuntimeException
```
Comment (malukenho) [inline src/Exception/ExtensionNotLoaded.php]: Should be removed. Use composer to check the dependencies instead

File: src/MySQLEventStore.php [hunk 2]
```diff
@@ -0,0 +1,374 @@
...
+        ActionEventEmitter $actionEventEmitter,
+        MessageFactory $messageFactory,
+        MessageConverter $messageConverter,
+        PDO $connection,
+        IndexingStrategy $indexingStrategy,
+        TableNameGeneratorStrategy $tableNameGeneratorStrategy,
+        int $loadBatchSize = 10000,
+        string $eventStreamsTable = 'event_streams'
+    ) {
+        if (! extension_loaded('pdo_mysql')) {
```
Comment (malukenho) [inline src/MySQLEventStore.php]: Not needs for this checker anymore

File: tests/AbstractPDOEventStoreTest.php [hunk 3]
```diff
@@ -0,0 +1,706 @@
...
+
+        $this->assertEquals('Prooph\Model\User', $stream->streamName()->toString());
+
+        $count = 0;
+        $lastEvent = null;
+        foreach ($stream->streamEvents() as $event) {
+            $count++;
+            $lastEvent = $event;
+        }
+        $this->assertEquals(2, $count);
```
Comment (malukenho) [inline tests/AbstractPDOEventStoreTest.php]: `assertCount()`

Comment (prolic) [inline tests/AbstractPDOEventStoreTest.php]: Yes. Tests still need a lot of Updates.

**Pull request merged**